core/domain/summary_services.py

```diff
@@ -218,12 +221,14 @@ def get_exp_metadata_dicts_matching_query(query_string, search_cursor, user_id):
         exp_services.get_exploration_ids_matching_query(
             query_string, cursor=search_cursor))
 
-    exploration_list = get_exploration_metadata_dicts(exp_ids, user_id)
+    exploration_list = get_exploration_metadata_dicts(
+        exp_ids, user_actions, user_id)
```

Comment:
We should have consistency: user_id always comes before user_actions. Though I realize you have a default arg issue...

Btw the third arg should be editor_user_id=user_id. (I know this was missed before, but arg names with default values always need to be explicitly written out.)